code/game/objects/structures/mirror.dm

```diff
@@ -167,3 +173,11 @@
 
 /obj/structure/mirror/magic/proc/curse(mob/living/user)
 	return
+
+/obj/structure/mirror/magic/nuclear
+	name = "M.A.G.I.C mirror"
+	desc = "The M.A.G.I.C mirror will let you change your species in a flash! Be careful, any implants (not biochips) in you will be destroyed on use."
```

Comment:
this part worries me, Gamma/Red ERTs will end up destroying their nutriment pumps or combat implants a lot b/c of this

Reply:
Only nukies have the mirror!

It warns them on examine and on use

Reply:
ohhh, that makes a lot more sense